src/main.c

```diff
@@ -0,0 +1,24 @@
+#include <pebble.h>
+
+#include "modules/comm.h"
+#include "modules/data.h"
+#include "windows/main_window.h"
+
+static void init() {
+  comm_init();
+
+  const int array_size = 16;
+  data_init(array_size);
+
+  main_window_push();
+}
+
+static void deinit() {
+  data_deinit();
```

Comment:
there should be a `comm_deinit()` or comment saying not required (since we push the practice of "anything you init, you need to deinit")